app/furniture/marketplace/cart.rb

```diff
@@ -12,7 +12,10 @@ class Cart < Record
 
     belongs_to :shopper, inverse_of: :carts
 
-    belongs_to :delivery_area, inverse_of: :carts, optional: true
+    belongs_to :delivery_area, inverse_of: :carts, class_name: "::Marketplace::DeliveryArea"
+    def cart_delivery_area
+      DeliveryArea.new(cart: self, delivery_area: delivery_area)
+    end
```

Comment:
I don't like this name collision... It's also weird that I have to make a pseudo-join table just so I can follow the Rails RESTful routing...

Reply:
I murderd it!

Reply:
Eggcellent murder.